link.sh

```diff
@@ -0,0 +1,51 @@
+
+#!/bin/bash
+BLUE='\033[0;34m'
+BBLUE='\033[1;34m' #BOLD
+PURPLE='\033[0;35m'
+RED='\033[0;31m'
+BRED='\033[1;31m' #BOLD
+GREEN='\033[0;32m'
+BGREEN='\033[1;32m' #BOLD
+GRAY='\033[1;30m'
+NC='\033[0m' # No Color
+
+echo -en "${BLUE}Installing @pxblue/highcharts dependencies...\n${NC}"
```

Comment:
Try to simplify these instructions a little bit (you can combine the angular and react linking steps together...see the changes I made in the chartjs script)